### PR TITLE
Business/jpa

### PR DESCRIPTION
**Pull request by sandyl289**
- Removed unused Umple code for Business.java class 
- Added JPA tags for Business class
- Created a Business CRUD repo (I only added "findBusinessByName(String name);")

### CODE REVIEW

File: RepairShop-Backend/src/main/java/ca/mcgill/ecse321/repairshop/model/Business.java [hunk 1]
```diff
@@ -1,242 +1,133 @@
...
     }
 
+    ///////////////////////////////////////////////////////////////
+
+/** BUSINESS ASSOCIATIONS **/
+
+    /************** TIMESLOT Vacations ****************/
+    // Business 0..1-->* TimeSlots (vacations)
+    //CacadeTupe.ALL: If delete Business, we want to delete ALL related TimeSlot vacations
+
```
Comment (JDziewonski98) [inline RepairShop-Backend/src/main/java/ca/mcgill/ecse321/repairshop/model/Business.java:105]: nice

File: RepairShop-Backend/src/main/java/ca/mcgill/ecse321/repairshop/model/Business.java [hunk 2]
```diff
@@ -1,242 +1,133 @@
...
+    private List<TimeSlot> vacations;
+
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "business")
```
Comment (JDziewonski98) [inline RepairShop-Backend/src/main/java/ca/mcgill/ecse321/repairshop/model/Business.java]: I don't think we need mappedBy here? I don't fully understand it but I think it's not necessary here in a unidirectional relationship

**Review by JDziewonski98 — APPROVED** [review]
Left one comment but otherwise looks good

File: RepairShop-Backend/src/main/java/ca/mcgill/ecse321/repairshop/model/Business.java (hunk 2 above)
Comment (sandyl289) [reply RepairShop-Backend/src/main/java/ca/mcgill/ecse321/repairshop/model/Business.java]: Thanks! I'll remove it!

**Pull request merged**